packages/data/src/world/oot/water_temple.yml

```diff
@@ -1,7 +1,7 @@
 "Water Temple":
   dungeon: Water
   exits:
-    "Lake Hylia": "true"
+    "Lake Hylia Near Water Temple": "true"
     "Water Temple Main": "true"
```

Comment:
While Water Temple is not being fully touched by this PR, it should probably still be mentioned you need to be able to swim or sink to both get out or access the main area of the temple, but it is not necessary to address for this PR.